deploy/prometheus/create-prometheus.sh

```diff
@@ -2,28 +2,31 @@
 
 echo "External value for \$PROMETHEUS_NAMESPACE=$PROMETHEUS_NAMESPACE"
 echo "External value for \$OPERATOR_NAMESPACE=$OPERATOR_NAMESPACE"
+echo "External value for \$BRANCH=$BRANCH"
```

Comment:
Why we need a branch here? For external components we do not need any branches and just to make sure the master version has correct manifests etc.

Reply:
please look at Line 59 https://github.com/Altinity/clickhouse-operator/blob/prometheus-operator-hotfix/deploy/prometheus/create-prometheus.sh#L59

I use tested this script with the following command (see https://github.com/Altinity/clickhouse-operator/blob/master/docs/prometheus_setup.md)

``` 
export BRANCH=prometheus-operator-hotfix
export PROMETHEUS_NAMESPACE=${PROMETHEUS_NAMESPACE:-prometheus}
curl -sL https://raw.githubusercontent.com/Altinity/clickhouse-operator/${BRANCH}/deploy/prometheus/create-prometheus.sh | bash
```

we will always use default value `master` for $BRANCH variable
but these changes allow us more flexibility

Reply:
i removed ${BRANCH} usage from hotfix